src/backends/caffe/caffelib.cc

```diff
@@ -1823,6 +1890,11 @@ namespace dd
 	  ad_res.add("regression",_regression);
 	if (inputc._ctc)
 	  ad_res.add("net_meas",true);
+       if (typeid(inputc) == typeid(CSVTSCaffeInputFileConn))
+         {
+           ad_res.add("timeserie",true);
+           ad_res.add("timeseries",nout);
```

Comment:
Do we need a special name for the output ? If it's a vector for instance, should be reuse names we use elsewhere, typically `vals` for an array of float/double ? Or is `nout` a special object, e.g. with timestamps ?

Reply:
it is a vector of vectors, [num_outputs * length of timeseries] so it seemed that a different name may avoid confusion